monitoring/sensu_check.py

```diff
@@ -257,6 +263,36 @@ def sensu_check(module, path, name, state='present', backup=False):
                     changed = True
                     reasons.append('`{opt}\' was removed'.format(opt=opt))
 
+        if module.params['custom']:
+          # Convert to json
+          try:
+            custom_params = ast.literal_eval(module.params['custom'])
```

Comment:
The python `dict` notation is not the same as the JSON notation. You should really use the JSON module here.